prs/ChicagoStyleSausagePizza.ts

```diff
@@ -0,0 +1,13 @@
+class ChicagoStyleSausagePizza extends Pizza {
```

Comment:
I would recommend including your access modifier to be explicit about the visibility of this class vs. expecting the developer to know the defaults for typescript.